week2/Week2/src/week2/EvenOrOdd.java

```diff
@@ -0,0 +1,32 @@
+
+package week2;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author sara
+ */
+public class EvenOrOdd {
+
+    public static void printNum(int number){
```

Comment:
Use descriptive name to clarify the purpose of a method
ex : printEvenOrOdd(int number)